lib/quartz/exceptions.rb

```diff
@@ -0,0 +1,10 @@
+module Quartz
+  class Exception < StandardError
+  end
+
+  class BadConfig < Exception
```

Comment:
I would prefer the name `ConfigError`. The message can then provide details as to why it's bad.